src/sagemaker/xgboost/estimator.py

```diff
@@ -38,7 +38,8 @@ class XGBoost(Framework):
     It can also handle training using customer provided XGBoost entry point script.
     """
 
-    _framework_name = defaults.XGBOOST_NAME
+    # _framework_name = defaults.XGBOOST_NAME
+    _framework_name: str = defaults.XGBOOST_NAME
```

Comment:
let's introduce type annotations outside of this PR

Reply:
will do! changed it to previous one